### PR TITLE
Fix map preview not extended to full view

### PR DESCRIPTION
**Pull request by kosmydel**
### Fixed Issues

$ https://github.com/Expensify/App/issues/28911
$ https://github.com/Expensify/App/pull/26307#discussion_r1348040334
PROPOSAL: N/A

### Tests

- [ ] Verify that no errors appear in the JS console
- [ ] Click Request Money
- [ ] Go to the distance tab
- [ ] Check the map appearance (verify border-radius, paddings, border-width, etc.)
- [ ] Select two random stops
- [ ] Press Next, select a workspace
- [ ] Check the map appearance (verify as above)
- [ ] **Do the same in the offline mode (so the Pending map view is displayed).**

### QA Steps

- [ ] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

Screenshot 2023-10-10 at 11 47 28
Screenshot 2023-10-10 at 11 48 26
Screenshot 2023-10-10 at 11 48 16
Screenshot 2023-10-10 at 11 47 57

**OLD:**

Screenshot 2023-10-05 at 16 32 42
Screenshot 2023-10-05 at 16 33 03

Screenshot 2023-10-05 at 23 57 46

</details>

<details>
<summary>Mobile Web - Chrome</summary>

**OLD:**

Screenshot 2023-10-05 at 16 43 54

Screenshot 2023-10-05 at 16 44 04

</details>

<details>
Screenshot 2023-10-10 at 11 50 46
Screenshot 2023-10-10 at 11 51 16
Screenshot 2023-10-10 at 11 51 20

<summary>Mobile Web - Safari</summary>
Screenshot 2023-10-10 at 11 50 41

**OLD:**

Screenshot 2023-10-05 at 16 44 12

Screenshot 2023-10-06 at 00 00 01

</details>

<details>
<summary>Desktop</summary>

**OLD:**

Screenshot 2023-10-05 at 16 37 46

Screenshot 2023-10-05 at 16 37 43

</details>

<details>
<summary>iOS</summary>

Screenshot 2023-10-10 at 11 49 19
Screenshot 2023-10-10 at 11 49 28
Screenshot 2023-10-10 at 11 49 38
Screenshot 2023-10-10 at 11 49 43

**OLD:**

Screenshot 2023-10-05 at 16 39 49

Screenshot 2023-10-05 at 16 40 00

Screenshot 2023-10-05 at 23 58 39

</details>

<details>
<summary>Android</summary>

Screenshot 2023-10-10 at 11 53 23
Screenshot 2023-10-10 at 11 54 13
Screenshot 2023-10-10 at 11 54 25
Screenshot 2023-10-10 at 11 54 35

**OLD:**

Screenshot 2023-10-05 at 16 40 56

Screenshot 2023-10-05 at 16 40 36

</details>

### CODE REVIEW

**Comment by situchan** [comment]
@jjcoffee I'll be reviewing this as this stemmed from the PR I reviewed

**Comment by shawnborton** [comment]
That looks good to me. I thought we also decided not to disable the submit button for the form submission. Do we know if that is being handled anywhere? cc @JmillsExpensify @trjExpensify @tgolen

**Comment by shawnborton** [comment]
@kosmydel can you also share what this looks like when we have say 5-6 waypoints?

**Comment by situchan** [comment]
> That looks good to me. I thought we also decided not to disable the submit button for the form submission. Do we know if that is being handled anywhere? cc @JmillsExpensify @trjExpensify @tgolen

yes - https://github.com/Expensify/App/issues/27045

**Comment by kosmydel** [comment]
> @kosmydel can you also share what this looks like when we have say 5-6 waypoints?

Sure @shawnborton, here are some screens (I will add them to the description as well):

<details>
<summary>Web</summary>

Screenshot 2023-10-05 at 23 57 46


</details>

<details>
<summary>iOS</summary>

Screenshot 2023-10-05 at 23 58 39


</details>

<details>
<summary>mWeb - iOS</summary>

Screenshot 2023-10-06 at 00 00 01


</details>

**Comment by shawnborton** [comment]
Nice, those look good. Can you show me what it looks like when you scroll down, too?

**Comment by kosmydel** [comment]
I've added the cleanup change mentioned [here](https://github.com/Expensify/App/pull/26307#discussion_r1348040334) to this PR.

--- 

> Nice, those look good. Can you show me what it looks like when you scroll down, too?

Sure @shawnborton!

<details>
<summary>Web</summary>


https://github.com/Expensify/App/assets/104823336/fb9341f8-5f92-49bf-b9fc-cdd3feb2fd96



</details>

<details>
<summary>iOS</summary>


https://github.com/Expensify/App/assets/104823336/7b918ca7-52b5-4aac-a806-687b0a7f4ace


</details>

<details>
<summary>mWeb - iOS</summary>


https://github.com/Expensify/App/assets/104823336/25560d81-d3ca-4763-bece-887cfd5b6f4c



</details>

<details>
<summary>mWeb - Android</summary>




https://github.com/Expensify/App/assets/104823336/5a2a4346-04a9-4a87-a10d-2f0509c881e5




</details>

<details>
<summary>Android</summary>


https://github.com/Expensify/App/assets/104823336/37fafa73-8001-4c23-ac1f-94079c74ac72



</details>

File: src/components/DistanceRequest/index.js
Comment (kosmydel) [inline src/components/DistanceRequest/index.js:234]: This is a cleanup change, discussed [here](https://github.com/Expensify/App/pull/26307#discussion_r1347867780).

**Comment by shawnborton** [comment]
Nice, that looks good to me! The only minor thing here is that at some point, the map area became skinnier than we wanted. cc @neil-marcellini as I know you have noticed this too. Basically it should look like this, which should match the width we are using for the camera area of the Scan page:
CleanShot 2023-10-09 at 08 30 47@2x

**Comment by kosmydel** [comment]
@shawnborton  I changed `padding` from `16px` to `8px`.

Here is how it looks:

<details>
<summary>Web</summary>

Screenshot 2023-10-09 at 15 02 05


</details>

<details>
<summary>iOS</summary>

Screenshot 2023-10-09 at 14 59 51

</details>

<details>
<summary>mWeb - iOS</summary>

Screenshot 2023-10-09 at 15 01 41


</details>

**Comment by shawnborton** [comment]
Nice! Can you confirm that's exactly what we have on the Scan page, as well as the same border radius we're using there? Thanks!

**Comment by kosmydel** [comment]
> Nice! Can you confirm that's exactly what we have on the Scan page, as well as the same border radius we're using there? Thanks!

I will check it tomorrow, as I'm finishing my work for today.

File: src/components/DistanceRequest/index.js
Comment (tgolen) [reply src/components/DistanceRequest/index.js:234]: Thank you!

**Comment by kosmydel** [comment]
Hey @shawnborton, I've updated the PR. I've added new QA steps and new screenshots. I had to make some more changes, because of how the Scan page handled the radius. Now it should look exactly as on the Scan page.

<details>
<summary>Web</summary>

Screenshot 2023-10-10 at 11 47 28
Screenshot 2023-10-10 at 11 48 26
Screenshot 2023-10-10 at 11 48 16
Screenshot 2023-10-10 at 11 47 57

</details>

<details>
<summary>Mobile Web - Safari</summary>

Screenshot 2023-10-10 at 11 50 46

Screenshot 2023-10-10 at 11 51 16

Screenshot 2023-10-10 at 11 51 20

Screenshot 2023-10-10 at 11 50 41

</details>

<details>
<summary>iOS</summary>

Screenshot 2023-10-10 at 11 49 19
Screenshot 2023-10-10 at 11 49 28
Screenshot 2023-10-10 at 11 49 38
Screenshot 2023-10-10 at 11 49 43
</details>

<details>
<summary>Android</summary>

Screenshot 2023-10-10 at 11 53 23
Screenshot 2023-10-10 at 11 54 13
Screenshot 2023-10-10 at 11 54 25
Screenshot 2023-10-10 at 11 54 35

</details>

**Review by tgolen — CHANGES_REQUESTED** [review]

**Comment by jjcoffee** [comment]
Can someone remove me as a reviewer and add @situchan instead, per [this comment](https://github.com/Expensify/App/pull/28922#issuecomment-1749151806)? Thanks!

**Comment by neil-marcellini** [comment]
Tagging in Rocio for review too since she's managing the issue.

**Comment by neil-marcellini** [comment]
Ah wait never mind that will happen after C+ approves.

**Comment by situchan** [comment]
reviewing this tomorrow

File: src/styles/styles.js
Comment (kosmydel) [reply src/styles/styles.js:3695]: Sure! I've moved it

File: src/styles/styles.js
Comment (kosmydel) [reply src/styles/styles.js:3700]: That's right! I've removed other styles while developing, and didn't catch that!

**Comment by kosmydel** [comment]
I've refactored it. Here is a video presenting both the map and its pending view:


<details>
<summary>
Video
</summary>

https://github.com/Expensify/App/assets/104823336/4bf58f16-0faf-4eb6-aac0-7e381d352f43

</details>

**Review by tgolen — DISMISSED** [review]
Great, thanks for refactoring those styles a little.

**Comment by tgolen** [comment]
There are some conflicts on here now. @situchan this is waiting on your final review.

**Comment by situchan** [comment]
I will checklist once conflict resolved

**Comment by kosmydel** [comment]
Conflicts resolved, and quickly tested it

cc @situchan

**Comment by pecanoro** [comment]
@situchan can you do a review as soon as possible? It's a high priority issue and it has been waiting for 3 days.

**Comment by situchan** [comment]
reviewing now

**Comment by situchan** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>


https://github.com/Expensify/App/assets/108292595/e1dc0606-4f3a-4e7f-a6fc-d6048402e203



</details>

<details>
<summary>Mobile Web - Chrome</summary>


https://github.com/Expensify/App/assets/108292595/027aa315-1f6b-4ce7-955f-ba73d69ddc1f



</details>

<details>
<summary>Mobile Web - Safari</summary>


https://github.com/Expensify/App/assets/108292595/20f35558-d948-417c-ba18-2b55023f46c8



</details>

<details>
<summary>Desktop</summary>


https://github.com/Expensify/App/assets/108292595/1cf5ff75-fc5d-4636-a866-daef1eb618e3



</details>

<details>
<summary>iOS</summary>


https://github.com/Expensify/App/assets/108292595/09251db4-e3df-4bba-84d9-ad6f2565fd88



</details>

<details>
<summary>Android</summary>


https://github.com/Expensify/App/assets/108292595/5257bb45-e30f-47a6-aecc-2314962e6f05



</details>

**Review by situchan — APPROVED** [review]
Pulled main into this branch locally and tests well (some known bugs are gone).

**Review by blazejkustra — APPROVED** [review]
LGTM, code looks good to me!

**Review by tgolen — APPROVED** [review]

**Pull request merged**
